Library/Homebrew/style.rb

```diff
@@ -172,8 +172,6 @@ def run_shellcheck(files, output_type)
         "--enable=all",
         "--external-sources",
         "--source-path=#{HOMEBREW_LIBRARY}",
```

Comment:
Maybe this can be removed (but np if not)

Reply:
Set `source-path=Library` in `.shellcheckrc` only add path `./Library` (under current working directory) to search path.

```
$ shellcheck bin/brew  # at HOMEBREW_REPOSITORY and get empty output

$ cd bin
$ shellcheck brew  # add `./Library` to search path but do not exist

In brew line 139:
  source "${HOMEBREW_LIBRARY}/Homebrew/brew.sh"
         ^-- SC1091 (info): Not following: ./Homebrew/brew.sh: openBinaryFile: does not exist (No such file or directory)

For more information:
  https://www.shellcheck.net/wiki/SC1091 -- Not following: ./Homebrew/brew.sh...

$ .brew style brew  # get empty output, since we add `--source-path=${HOMEBREW_LIBRARY}` in `brew style`
```